### PR TITLE
Prep for pub

### PR DESCRIPTION
**Pull request by saulakravitz**
1) Fixes to index.html
2) changes to version in sushi-config and publlication-request
3) Updated data dictionary and differential data dictionary
4) fixes to references to mCODE to use link to version cited in sushi-config, rather than a hardcoded version.

### CODE REVIEW

File: input/images/data-dictionary/~$CodexRTDataDictionary-STU2-vs-STU1.xlsx
Comment (mvonsieb) [inline input/images/data-dictionary/~$CodexRTDataDictionary-STU2-vs-STU1.xlsx:1]: Is this file intentionally included?

File: input/pagecontent/history.md [hunk 1]
```diff
@@ -3,12 +3,15 @@ In late 2020, the American Society for Radiation Oncology (ASTRO) and the Americ
...
 
-In support of this collaboration, radiotherapy profiles were added to [mCODE STU3](https://hl7.org/fhir/us/mcode/STU3), [Radiotherapy Course Summary](https://hl7.org/fhir/us/mcode/STU3/StructureDefinition-mcode-radiotherapy-course-summary.html) and [Radiotherapy Volume](https://hl7.org/fhir/us/mcode/STU3/StructureDefinition-mcode-radiotherapy-volume.html), as well as other value sets and extensions required to represent a radiotherapy treatment summary.
+In support of this collaboration, radiotherapy profiles were added to [mCODE], [Radiotherapy Course Summary][[mCODE Radiotherapy Course Summary] and [Radiotherapy Volume][mCODE Radiotherapy Volume], as well as other value sets and extensions required to represent a radiotherapy treatment summary.
```
Comment (mvonsieb) [inline input/pagecontent/history.md]: One bracket too much [[

File: input/images/data-dictionary/CodexRTDataDictionary-STU2.xlsx
Comment (mvonsieb) [inline input/images/data-dictionary/CodexRTDataDictionary-STU2.xlsx:1]: In the data dict, the version still says 2.0.0-ballot

File: input/pagecontent/change_log.md [hunk 2]
```diff
@@ -1,90 +1,25 @@
...
-* New SNOMEDCT codes requested by the authors have replaced local codes as they have been issued.
-
-### Capability Statement Changes
-* Added a server [Capability Statement](CapabilityStatement-CodexRTServerCapabilityStatement.html) based on XRTS requirements.  Included credits for rendering.
-
-### Added Data Dictionary
-
-* Added a [data dictionary](dictionary.html) with the same format as the mCODE data dictionary.
+### CodeX Radiation Therapy STU2 Changes
+* Changed mCODE dependency to [STU4]({{site.data.fhir.ver.fhirmcode}}/index.html)
```
Comment (mvonsieb) [inline input/pagecontent/change_log.md:2]: Shouldn't this be a versioned link?

**Review by mvonsieb — APPROVED** [review]
Looks good thanks. See comments.

File: input/images/data-dictionary/~$CodexRTDataDictionary-STU2-vs-STU1.xlsx
Comment (saulakravitz) [reply input/images/data-dictionary/~$CodexRTDataDictionary-STU2-vs-STU1.xlsx:1]: No...it is garbage... will remove.

File: input/pagecontent/change_log.md (hunk 2 above)
Comment (saulakravitz) [reply input/pagecontent/change_log.md:2]: It is versioned.   The {{site.data...}} mechanism is based on the sushi-config.yaml dependency for mCODE.

File: input/images/data-dictionary/CodexRTDataDictionary-STU2.xlsx
Comment (saulakravitz) [reply input/images/data-dictionary/CodexRTDataDictionary-STU2.xlsx:1]: I'm not seeing that...

**Pull request merged**